apps/modernization-ui/src/apps/page-builder/page/management/edit/subsection/Subsection.tsx

```diff
@@ -1,25 +1,39 @@
-import { PagesSubSection } from 'apps/page-builder/generated';
+import { PageQuestionControllerService, PagesSubSection } from 'apps/page-builder/generated';
 import { SubsectionHeader } from './SubsectionHeader';
 import styles from './subsection.module.scss';
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import { Question } from '../question/Question';
+import { useAlert } from '../../../../../../alert';
+import { useParams } from 'react-router-dom';
+import { UserContext } from '../../../../../../providers/UserContext';
 
 type Props = {
     subsection: PagesSubSection;
     onAddQuestion: () => void;
 };
 export const Subsection = ({ subsection, onAddQuestion }: Props) => {
     const [isExpanded, setIsExpanded] = useState<boolean>(true);
+    const { state } = useContext(UserContext);
 
     const handleExpandedChange = (expanded: boolean) => {
         setIsExpanded(expanded);
     };
 
+    const token = `Bearer ${state.getToken()}`;
```

Comment:
Suggestion (non-blocking): There is a new `authorization()` method that handles creating the token for you.